docs/content/docs/guide/privaterepo.md

```diff
@@ -4,36 +4,34 @@ weight: 7
 ---
 # Private repositories
 
-Pipelines as Code enables the use of private repositories by creating or
```

Comment:
cc @Bibz87  for the rephrasing i meant in other PR review

Reply:
Thanks, I'll write better doc next time! 😄